src/PositionManager.sol

```diff
@@ -206,6 +220,22 @@ contract PositionManager is
                 ) = params.decodeMintParams();
                 _mint(poolKey, tickLower, tickUpper, liquidity, amount0Max, amount1Max, _mapRecipient(owner), hookData);
                 return;
+            } else if (action == Actions.MINT_POSITION_FROM_AMOUNTS) {
+                (
+                    PoolKey calldata poolKey,
+                    int24 tickLower,
+                    int24 tickUpper,
+                    uint128 amount0,
```

Comment:
now that i think about it im wondering if theres ever a time that `amount0` and `amount1` would ever not just be OPEN_DELTA? I dont really see a usecase aside from that?
so could even potentially just remove those parameters and just always look up open deltas and then add that much liquidity?